tasks/configure.yml

```diff
@@ -4,9 +4,9 @@
     src: "alert.rules.j2"
     dest: "{{ prometheus_config_dir }}/rules/ansible_managed.rules"
     owner: root
-    group: prometheus
+    group: "{% if prometheus_docker_install %} 65534 {% else %} prometheus {% endif %}"
     mode: 0640
-    validate: "/usr/local/bin/promtool check rules %s"
+    validate: "{{ prometheus_promtool_exec }} check rules %s"
```

Comment:
`prometheus_promtool_exec` should be set in `vars/main.yml` and should be prefixed with `__`.

Reply:
Got it !

just to be clear :
- I set this variables in vars with `/usr/local/bin/promtool` as value and override it if the prometheus_docker_install is triggered right  (in preflight) ?

Still i'm curious, what is the need to prefix this variable with __ ?

Reply:
`__` prefix is a common thing in python world to say that this variable is "private"